blockchain/checkpoints.go

```diff
@@ -98,7 +98,12 @@ func (b *BlockChain) maybeUpdateMostRecentCheckpoint(node *blockNode) {
 func isNonstandardTransaction(tx *dcrutil.Tx) bool {
 	// Check all of the output public key scripts for non-standard scripts.
 	for _, txOut := range tx.MsgTx().TxOut {
-		scriptClass := txscript.GetScriptClass(txOut.Version, txOut.PkScript)
+		// It is ok to hard code isTreasuryEnabled to false here since
+		// there is no way the chain is going to have treasury
+		// transactions in it prior to activation since they are stake
+		// transactions and would be rejected as unknown types.
+		scriptClass := txscript.GetScriptClass(txOut.Version,
+			txOut.PkScript, false)
```

Comment:
Looks like you set it to false (and comment too) instead of true as the original review comment indicates.

I'm fairly certain this should be `true` so that it works with the new treasury transactions once they are permitted into the chain.  It is safe to do because the consensus rules will reject any such stake transactions before the agenda is active.  The comment is otherwise correct.

Reply:
Makes sense. Fix pushed.